JS/script.js

```diff
@@ -314,3 +314,22 @@ seeProject.forEach((links) => {
     popupCont.classList.add('active');
   });
 });
+
+const submitButton = document.querySelector('#submit-button');
+function validate() {
+  const text = document.getElementById('email').value;
+  const validator = /^([a-z\d-]+)@([a-z\d-]+)\.([a-z]{2,8})(\.[a-z]{2,8})?$/;
+  document.forms[0].onsubmit = function email(e) {
+    if (!(validator.test(text))) {
+      document.getElementById('error-msg').innerHTML = 'Please use lowercase to enter valid email address!';
+      document.getElementById('error-msg').style.visibility = 'visible';
+      document.getElementById('error-msg').style.color = 'white';
+      document.getElementById('error-msg').style.background = 'red';
+      document.getElementById('error-msg').style.textAlign = 'center';
```

Comment:
- [ ] [ OPTIONAL ] Please consider reducing the reb bg color, you can keep the style but play with the red color to make it a bit less bright because it does not really match the color of your page you see

<img width="482" alt="1" src="https://user-images.githubusercontent.com/77843050/211030637-4f90ab3e-4f46-4ed7-99db-84cebec3d44b.png">